doc/api/errors.md

```diff
@@ -1212,13 +1212,13 @@ for strict compliance with the API specification (which in some cases may accept
 `func(undefined)` and `func()` are treated identically, and the
 [`ERR_INVALID_ARG_TYPE`][] error code may be used instead.
 
-<a id="ERR_MISSING_DYNAMIC_INSTANTIATE_HOOK"></a>
-### ERR_MISSING_DYNAMIC_INSTANTIATE_HOOK
+<a id="ERR_MISSING_DYNAMIC_TRANSLATE_HOOK"></a>
+### ERR_MISSING_DYNAMIC_TRANSLATE_HOOK
```

Comment:
I understand you say the name change here of "instantiate" to "translate" is inspired by the loader spec, but from the beginning to the current WhatWG loader spec, translate has always been to donate a source to source translation, while instantiate has been used to indicate the interpretation of a module source.

Reply:
this is a translation hook, even if the user doesn't directly appear so to the user.

Reply:
@devsnek while that may be true from an implementation perspective, the code at https://github.com/nodejs/node/blob/master/lib/internal/loader/ModuleWrap.js#L17 is an implementation detail. This API could equally well have been exposed by v8 through a custom module API without needing the module wrapper (for example there was early work on a spec for reflective module records - https://gist.github.com/dherman/fbf3077a2781df74b6d8 which would have been part of loader spec).

In addition a true translation hook might be provided through a separate loader hook in future (eg `export function translate` on the loader), which would be separate to this API, so I worry about conflating meanings here.

Reply:
well at this point instantiation has already happened, maybe "dynamic bind" or "dynamic map"?

Reply:
Instantiation hasn't already happened at this point (instantiation happens after translation) - instantiation needs as an input the list of module exports, which is what this hook provides. What specifically do you find is misleading about the "instantiate" term?